shared/teams/render.js

```diff
@@ -85,11 +120,12 @@ const Render = (props: Props) => (
         ...globalStyles.flexBoxColumn,
         alignItems: 'center',
         marginBottom: globalMargins.tiny,
-        marginLeft: globalMargins.large,
-        marginRight: globalMargins.large,
+        marginLeft: globalMargins.tiny,
```

Comment:
see my banner pr, the margins likely shouldn't be here, scrolling gets messed up in mobile

Reply:
done, changed to padding in TeamList